content/webapp/app.ts

```diff
@@ -20,6 +19,7 @@ import {
 } from '@weco/common/services/prismic/hardcoded-id';
 import { Periods } from '@weco/common/model/periods';
 import linkResolver from './services/prismic/link-resolver';
+import * as prismic from 'prismic-client-beta';
```

Comment:
is the plan to move everything to `prismic-client-beta`, then switch all of those out for v6 of `@prismicio/client` (which presumably has the same API)?

Reply:
Yup!